plugins/modules/sysctl.py

```diff
@@ -46,16 +46,29 @@
               C(sysctl_file) is updated.
         type: bool
         default: 'yes'
-    sysctl_file:
+    path:
```

Comment:
why this ? Is it related to this PR ?

Generally speaking, you should avoid to add aliases. I would not be surprised if other reviewers complain about this. On the other hand, I personally think that almost all modules handling files should have a `path` option to point to the file, and all other should probably have a `name` :D

Anyway, as far as I know, it is not required to name the file by `path` to make use of *file common arguments*. You can pass in argument `path=sysctl_file` to `module.load_file_common_arguments()`. In that case, the module will not return all the (plethora) stat results associated to the `path` option, all those we don't have to care about here (a lot of *user readable*, *user writable*, *user executable*, *group readable* and so on, for example). In other words, if the returned key is called `path`, we get a pretty more verbose result than if it is called `sysctl_file`.

You can take a look here for a real-life example of a target file not called `path`: https://github.com/ansible-collections/community.general/blob/549dfaae6415999755f2ca722b39e8c314c67aa1/plugins/modules/system/java_keystore.py#L263-L271

Reply:
Another example, where the option is named `path`, but the module is aware of the size of the file in its results, so it ensures *file common arguments* are applied to (or gathered from) the actual file rather than to a symlink pointing to this file:
https://github.com/ansible-collections/community.general/blob/549dfaae6415999755f2ca722b39e8c314c67aa1/plugins/modules/files/filesize.py#L467-L477

Reply:
I really appreciate your kind advice!
As you pointed out, this was a pretty drastic idea. So I would undo it and fix it to take advantage of the path option. :D

Reply:
oh, 2.9 seems not to support `path`. I'll try something.

https://docs.ansible.com/ansible/2.9/reference_appendices/module_utils.html#ansible.module_utils.basic.AnsibleModule.load_file_common_arguments

Reply:
done :)